### PR TITLE
Add DB insertion functions

### PR DESCRIPTION
**Pull request by erikd**
This includes tests that are almost certain to fail with DB connections issues.

### CODE REVIEW

File: cardano-explorer-core/test/test-db.hs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+tests :: TestTree
+tests =
+  testGroup "Database"
+    [ testCase "Migration" testMigration
+    ]
+
+testMigration :: IO ()
+testMigration =
+  runMigrations True (PGPassFile "../config/pgpass") (MigrationDir "../schema") (LogFileDir "..")
```
Comment (dcoutts) [inline cardano-explorer-core/test/test-db.hs]: Is `config/pgpass` something that should exist in the repo, or does one have to make one to run the test?

**Review by dcoutts — APPROVED** [review]
LGTM.

File: cardano-explorer-core/test/test-db.hs (hunk 1 above)
Comment (erikd) [reply cardano-explorer-core/test/test-db.hs]: Yeah, need to figure that out.

Comment (erikd) [reply cardano-explorer-core/test/test-db.hs]: It has been figured out. Subsequent commits on this branch add that file (without any credentials).

**Pull request merged**